### PR TITLE
Design-Nav_li_padding

### PR DESCRIPTION
**Pull request by jbeck007**
General CSS cleanup, style and script bundles, nav sub-menu li's centered and padding applied.

### CODE REVIEW

**Comment by jbeck007** [comment]
I centered the text in the sub-menu's. Let me know what you think. I am not married to it by any means.

File: GlowDotOrb/Content/css/main.css [hunk 1]
```diff
@@ -219,10 +205,6 @@ ul.sub-menu {
   color: #fff;
```
Comment (germanlph) [inline GlowDotOrb/Content/css/main.css]: Is the hover color one of the brand colors? Same question for the hover of nav bar links and for the border around the sub-menu box.

File: GlowDotOrb/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -26,7 +19,8 @@
 </head><!--/head-->
 <body>
     <header id="header">
-        
+
+
```
Comment (germanlph) [inline GlowDotOrb/Views/Shared/_Layout.cshtml]: Why all the empty space ;) ?

File: GlowDotOrb/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -94,19 +89,11 @@
                     
                         <li> @Html.Partial("_LoginPartial")</li>
                     </ul>
-                </div>
-
-
-                  
-                
-               
+                </div>
```
Comment (germanlph) [inline GlowDotOrb/Views/Shared/_Layout.cshtml]: Nicely done.

File: GlowDotOrb/Views/Shared/_Layout.cshtml (hunk 2 above)
Comment (jbeck007) [inline GlowDotOrb/Views/Shared/_Layout.cshtml]: You know this is my safe space.

File: GlowDotOrb/Content/css/main.css (hunk 1 above)
Comment (jbeck007) [inline GlowDotOrb/Content/css/main.css]: It is not. Good catch. I will change that.

**Pull request merged**